apps/judicial-system/api/src/app/modules/defendant/dto/updateDefendant.input.ts

```diff
@@ -136,4 +136,14 @@ export class UpdateDefendantInput {
   @IsOptional()
   @Field(() => String, { nullable: true })
   readonly alternativeServiceDescription?: string
+
+  @Allow()
+  @IsOptional()
+  @Field(() => String, { nullable: true })
+  readonly verdictForDefendant?: string
+
+  @Allow()
+  @IsOptional()
+  @Field(() => [String], { nullable: true })
+  readonly informationForDefendant?: string[]
```

Comment:
Please use an enum here. The enum should be stored in the `defendant.ts` file in the type lib and a GraphQL enum should be registered in `defendant.model.ts`. You can use `VerdictAppealDecision` as an example.

Please use a descriptive name for the enum and use upper case English names for each enum member as in the example.